ci-operator/step-registry/ipi/aws/post/ipi-aws-post-chain.yaml

```diff
@@ -4,5 +4,6 @@ chain:
   - ref: gather-aws-console
   - chain: ipi-deprovision
   - ref: aws-deprovision-users-and-policies
+  - ref: ipi-deprovision-aws-eip
```

Comment:
we could create a new chain for `ipi-deprovision-aws`, or `aws-deprovision`  to aggregate those new steps.